app/controllers/idv/hybrid_mobile/socure/document_capture_controller.rb

```diff
@@ -28,6 +28,9 @@ def show
           @document_response = document_response
           @url = document_response.dig(:data, :url)
 
+          # placeholder until we get an error page for url not being present
+          redirect_to idv_unavailable_url if @url.nil?
```

Comment:
`redirect_to` doesn't terminate the request immediately. Are we still wanting the next lines of code to be run if this condition matches?